io.lmms.LMMS.appdata.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 the authors -->
+<component type="desktop">
+ <id>io.lmms.LMMS</id>
+ <metadata_license>CC0</metadata_license>
+ <project_license>GPL-2.0+</project_license>
+ <name>LMMS</name>
+ <summary>A music production application</summary>
+ <description>
+   <p>LMMS is a music production application. It allows the creation
+   of loops, synthesizing and mixing sounds, arranging samples having
+   fun with your MIDI-keyboard and much more... It combines the
+   features of a tracker-/sequencer-program and those of powerful
+   synthesizers, samplers, effects etc. in a modern, user-friendly and
+   easy to use graphical user-interface.</p>
+ </description>
+ <content_rating type="oars-1.1" />
+ <launchable type="desktop-id">io.lmms.LMMS.desktop</launchable>
+ <translation type="qt">lmms</translation>
+ <screenshots>
+  <screenshot type="default">
+   <image>https://lmms.io/wiki/images/thumb/2/2d/Startup_LMMS_new.png/1200px-Startup_LMMS_new.png</image>
+   <caption>The main LMMS window</caption>
+  </screenshot>
+ </screenshots>
+ <url type="homepage">https://lmms.io/</url>
```

Comment:
More links would be appreciated:
 * `bugtracker` - https://github.com/LMMS/lmms/issues
 * `faq` - https://lmms.io/documentation/User_FAQ
 * `help` - https://lmms.io/documentation/
 * `translate` - https://github.com/LMMS/lmms/wiki/Creating-a-localization

See also:
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-url

Reply:
done